packages/eslint-plugin/lib/config/rules/jest-formatting.js

```diff
@@ -0,0 +1,16 @@
+// see https://github.com/dangreenisrael/eslint-plugin-jest-formatting#usage
+
+module.exports = {
+  // Require padding around afterAll blocks
```

Comment:
I like this one too https://github.com/dangreenisrael/eslint-plugin-jest-formatting/blob/HEAD/docs/rules/padding-around-expect-groups.md#padding-around-expect-groups

Reply:
Yeah I think that one comes as part of the `strict` config, and I only included those from the `recommended` config, so as to be less contentious. We can add it in too, but I suspect that'll be the one that people might be more likely to disable based on personal preference